src/VisualStudio/Core/Test/Diagnostics/ExternalDiagnosticUpdateSourceTests.vb

```diff
@@ -27,7 +27,7 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.Diagnostics
 
         <Fact>
         Public Async Function TestExternalDiagnostics_RaiseEvents() As Task
-            Using workspace = Await CSharpWorkspaceFactory.CreateWorkspaceFromFileAsync(String.Empty)
+            Using workspace = Await TestWorkspace.CreateCSharpAsync(String.Empty)
```

Comment:
Does `String.Empty` really need to be passed everytime? Seems like that parameter should be optional, or there should be another factory method (e.g. `TestWorkspace.CreateEmptyCSharpAsync(...)`).